.npmignore

```diff
@@ -1 +1,6 @@
 src/
+node_modules
+/npm-debug.log*
+.DS_Store
+lib/
```

Comment:
`lib/` is the compiled distributable. We definitely want this to remain in `npm` 😄